data/ratings/pass.ts

```diff
@@ -0,0 +1,24 @@
+import { RankDetail } from '../../services/elasticsearch'
+
+type Pass = {
+  score: number // should be a range between 0 => 1
+  pass: boolean
+}
+type PassFn = (score: RankDetail) => Pass
+
+const eq1: PassFn = (rankDetail: RankDetail) => {
```

Comment:
only real suggestion is that these names could be a bit more descriptive, and that we could set one as a default to avoid repetition in the data - looks good otherwise!

Reply:
More descriptive? Do you have an example?

re: default to 1 - do you suggest having `eq(n = 1)`?